src/time.cr

```diff
@@ -187,23 +406,71 @@ struct Time
     new(seconds: seconds, nanoseconds: nanosecond.to_i, location: location)
   end
 
+  # Creates a new `Time` instance representing the given date-time in UTC.
+  #
+  # ```
+  # time = Time.utc(2016, 2, 15, 10, 20, 30)
+  # time.to_s # => 2016-02-15 10:20:30 UTC
+  # ```
+  #
+  # Valid value ranges for the individual fields:
+  #
+  # * `year`: `1..9999`
+  # * `month`: `1..12`
+  # * `day`: `1` - `28`/`29`/`30`/`31` (depending on month and year)
+  # * `hour`: `0..23`
+  # * `minute`: `0..59`
+  # * `second`: `0..59`
+  # * `nanosecond`: `0..999_999_999`
+  #
+  # The time-of-day can be omitted and defaults to midnight (start of day):
+  #
+  # ```
+  # time = Time.utc(2016, 2, 15)
+  # time.to_s # => 2016-02-15 00:00:00 UTC
+  # ```
+  #
+  # Since UTC does not have any time zone transitions, each date-time is
+  # unambiguously resolved.
+  def self.utc(year : Int, month : Int, day : Int, hour : Int = 0, minute : Int = 0, second : Int = 0, *, nanosecond : Int = 0) : Time
+    new(year, month, day, hour, minute, second, nanosecond: nanosecond, location: Location::UTC)
+  end
+
+  # Creates a new `Time` instance that corresponds to the number of *seconds*
+  # and *nanoseconds* elapsed from epoch (`0001-01-01 00:00:00.0 UTC`)
+  # observed in *location*.
+  #
+  # Valid range for *seconds* is `0..315_537_897_599`.
+  # For *nanoseconds* it is `0..999_999_999`.
+  def initialize(*, @seconds : Int64, @nanoseconds : Int32, @location : Location)
+    unless 0 <= @nanoseconds < NANOSECONDS_PER_SECOND
```

Comment:
BTW does this method not range check `@seconds`???

Reply:
Nope. But #5786 is just waiting for a second review =)

Reply:
Nice pun ;-)